src/test/java/seedu/cafectrl/command/AddDishCommandTest.java

```diff
@@ -18,7 +18,7 @@ void execute_oneDishAdded_expectDishInMenu() {
         Menu menu = new Menu(menuItems);
         Ui ui = new Ui();
         //creating a dish
-        ingredients.add(new Ingredient("chicken", "100g"));
+        ingredients.add(new Ingredient("chicken",100, "g"));
```

Comment:
could you perhaps add a white space after the comma?

```suggestion
        ingredients.add(new Ingredient("chicken", 100, "g"));
```